whitelabelco/sofo-foods
# Cp/bug/brand page cleanup

### Description 💻

Resolves: #148 

### Implementation 💡

remove margin from Brands hero and reduces padding in following section

Reference ticket for bug demo

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- switch to mobile view
- click "Brands"
- confirm space between nav and hero is gone
- confirm space between hero and start of brands section matches comps

### Screenshots/Demos 📷
![Screenshot 2024-10-28 at 8 22 47 AM](https://github.com/user-attachments/assets/d0b88b9f-abde-4684-b622-79e39515eae1)




### Checklist ✅
